dotcom-rendering/scripts/deploy/build-riffraff-bundle.js

```diff
@@ -31,7 +31,13 @@ const target = path.resolve(__dirname, '../..', 'target');
 
 const copyCfn = () => {
 	log(' - copying cloudformation config');
-	return cpy([`cloudformation.yml`], path.resolve(target, 'frontend-cfn'));
+	return cpy(
+		[
+			'./cdk.out/DotcomRendering-CODE.template.json',
+			'./cdk.out/DotcomRendering-PROD.template.json',
+		],
+		path.resolve(target),
```

Comment:
Could we break this piece of work up to reduce the PR size, cognitive load on deploying it and maybe adding a little confidence and less scariness

e.g:
* building cloudformation with CDK
* deploying newly built cloudformation for code
* test
* deploying newly built cloudformation for prod
* test
* remove old cloudformation.yml

Reply:
Sounds good!

I think this will reduce the cognitive load of reviewing this PR, but deployment is always going to be a little scary - we're either using CDK or we're not